src/command_modules/azure-cli-acr/setup.py

```diff
@@ -34,7 +34,7 @@
     'azure-mgmt-resource==2.0.0',
     'azure-mgmt-storage==1.5.0',
     'azure-storage-blob==1.1.0',
-    'azure-mgmt-containerregistry==2.0.0',
+    'azure-mgmt-containerregistry==2.1.0',
```

Comment:
Can you update the version here? otherwise the CI won't succeed.

Reply:
Done.